k8s/gcp-pubsub/destroy-pubsub.sh

```diff
@@ -10,4 +10,6 @@ kubectl delete -f turbinia-autoscale-cpu.yaml
 kubectl delete -f turbinia-server-metrics-service.yaml 
 kubectl delete -f turbinia-worker-metrics-service.yaml 
 kubectl delete -f turbinia-worker.yaml 
-kubectl delete -f turbinia-server.yaml 
\ No newline at end of file
+kubectl delete -f turbinia-server.yaml
+kubectl delete -f turbinia-output-filestore.yaml 
+kubectl delete -f turbinia-output-claim-filestore.yaml 
```

Comment:
I think you need to switch the deletion order of the claim and filestore because if you delete the filestore first the claim will be hanging. So it should be the reverse of the setup order.

Reply:
Ah good catch/that makes sense, I went ahead and switched the order around